src/overlaybd/zfile/zfile.cpp

```diff
@@ -745,8 +786,7 @@ IFile *zfile_open_ro(IFile *file, bool verify, bool ownership) {
             auto res = file->fallocate(0, 0, -1);
```

Comment:
if `load_jump_table` fails, it will evict the whole file from cache and retry @shuaichang